project1/css/index.css

```diff
@@ -52,13 +49,124 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html, body {    
+    display: flex;
     font-family: 'Titillium Web', sans-serif;
+    align-content: center;
+    justify-content: center;
+    flex-wrap: wrap;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
-    margin-bottom: 10px;
 }
+
+.container {
+  width: 800px;
+  margin: 0 auto;
```

Comment:
You can omit `margin: 0 auto;` since you `body` is now a flex container with `justify-content: center;`